cognite_toolkit/_cdf_tk/run.py

```diff
@@ -8,7 +8,6 @@
 import subprocess
 import tempfile
 import time
-import venv
```

Comment:
pyodide does not have `venv`, so moved it to a local import.

Reply:
Yes, running local doesn't have meaning in a notebook :)